### DISCUSSION TITLE
Object Customizable Left Click Potential Bug

### DISCUSSION THREAD

**Discussion by erobert98** [runelite/runelite#14996] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/31151011/169634691-026f992b-368e-49b3-b2a1-8d356649cc70.png)

Ive swapped the setting of traps to check instead of reset and it wont swap the left click. Same issue for red traps that i want to dismantle rather than reset.

**Comment by Nightfirecat** [comment, marked as answer]
This is because the `Reset` option is enabled in the object swaps section. Disabling that will allow left-click customization to take priority.
